### ISSUE TITLE
sign-0.4.1 does not compile

### ISSUE THREAD

**Issue by peti** [msakai/sign#2]
Citing from http://hydra.cryp.to/build/1304217/log/raw:

```
[1 of 1] Compiling Data.Sign        ( src/Data/Sign.hs, dist/build/Data/Sign.o )

src/Data/Sign.hs:128:10:
    Overlapping instances for L.MeetSemiLattice (Set Sign)
      arising from the superclasses of an instance declaration
    Matching instances:
      instance Ord a => L.MeetSemiLattice (Set a)
        -- Defined in ‘Algebra.Lattice’
      instance L.MeetSemiLattice (Set Sign)
        -- Defined at src/Data/Sign.hs:123:10
    In the instance declaration for
      ‘L.BoundedMeetSemiLattice (Set Sign)’

src/Data/Sign.hs:131:10:
    Overlapping instances for L.Lattice (Set Sign)
      arising from the superclasses of an instance declaration
    Matching instances:
      instance Ord a => L.Lattice (Set a) -- Defined in ‘Algebra.Lattice’
      instance L.Lattice (Set Sign) -- Defined at src/Data/Sign.hs:126:10
    In the instance declaration for ‘L.BoundedLattice (Set Sign)’
```

**Comment by msakai** [comment]
Thank you for reporting that.
This is caused by the changes in lattices-1.4, and I'll release the fixed version soon.

**Comment by msakai** [comment]
I've fixed the problem with d7f1bccc8709184ccceb22ae5e206383532ee0f3 and uploaded a new version 0.4.2 to Hackage.
